Documentation/CRDs/Cluster/external-cluster/consumer-cluster.md

```diff
@@ -0,0 +1,73 @@
+# Commands on the K8s consumer cluster
+
+## Helm Installation
+
+To install with Helm, the rook cluster helm chart will configure the necessary resources for the external cluster with the example `values-external.yaml`.
+
+```console
+clusterNamespace=rook-ceph
+operatorNamespace=rook-ceph
+cd deploy/examples/charts/rook-ceph-cluster
+helm repo add rook-release https://charts.rook.io/release
+helm install --create-namespace --namespace $clusterNamespace rook-ceph rook-release/rook-ceph -f values.yaml
+helm install --create-namespace --namespace $clusterNamespace rook-ceph-cluster \
+--set operatorNamespace=$operatorNamespace rook-release/rook-ceph-cluster -f values-external.yaml
+```
+
+Skip the manifest installation section and continue with [Cluster Verification](#cluster-verification).
+
+## Manifest Installation
+
+If not installing with Helm, here are the steps to install with manifests.
+
+1. Deploy Rook, create [common.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common.yaml), [crds.yaml](https://github.com/rook/rook/blob/master/deploy/examples/crds.yaml) and [operator.yaml](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml) manifests.
+
+2. Create [common-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/external/common-external.yaml) and [cluster-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/external/cluster-external.yaml)
+
+# Import the Provider Data
```

Comment:
Only the first title on the page is typically level 1, so level 2 seems more standard here
```suggestion
## Import the Provider Data
```